vue-components/src/components/Popover.vue

```diff
@@ -179,5 +200,127 @@ $base: '.wikit-Popover';
 			transform-origin: bottom left;
 		}
 	}
+
+	&--bottom-center {
+		#{$base}__pointer {
+			inset-inline-start: calc(50% - #{$wikit-Popover-pointer-width} / 2);
+		}
+	}
+	&--bottom-right {
+		#{$base}__content-wrapper {
+			transform: translateX(0);
+		}
+		#{$base}__pointer {
+			inset-inline-start: $wikit-Popover-pointer-width / 2;
+		}
+	}
+	&--bottom-left {
+		#{$base}__content-wrapper {
+			transform: translateX(-100%);
+		}
+		#{$base}__pointer {
+			inset-inline-start: calc(100% - (#{$wikit-Popover-pointer-width} + #{$wikit-Popover-pointer-width} / 2));
+		}
+	}
+
+	&--top-center {
+		#{$base}__content-wrapper {
+			margin-block-start: -($wikit-Popover-distance + $wikit-Popover-pointer-height);
+			transform: translate(-50%, -100%);
+			inset-block-start: 0%;
+		}
+		#{$base}__pointer {
+			inset-block-start: 100%;
+		}
+
+		#{$base}__pointer::before {
+			inset-block-start: calc(-1 * (50% + #{$wikit-Popover-pointer-width} / 2));
+		}
+	}
+	&--top-right {
+		#{$base}__content-wrapper {
+			margin-block-start: -($wikit-Popover-distance + $wikit-Popover-pointer-height);
+			transform: translate(0, -100%);
+			inset-block-start: 0%;
+		}
+		#{$base}__pointer {
+			inset-block-start: 100%;
+			inset-inline-start: $wikit-Popover-pointer-width / 2;
+		}
+
+		#{$base}__pointer::before {
+			inset-block-start: calc(-1 * (50% + #{$wikit-Popover-pointer-width} / 2));
+		}
+	}
+	&--top-left {
+		#{$base}__content-wrapper {
+			margin-block-start: -($wikit-Popover-distance + $wikit-Popover-pointer-height);
+			transform: translate(-100%, -100%);
+			inset-block-start: 0%;
+		}
+		#{$base}__pointer {
+			inset-block-start: 100%;
+			inset-inline-start: calc(100% - (#{$wikit-Popover-pointer-width} + #{$wikit-Popover-pointer-width} / 2));
+		}
+
+		#{$base}__pointer::before {
+			inset-block-start: calc(-1 * (50% + #{$wikit-Popover-pointer-width} / 2));
+		}
+	}
+
+	&--right-center {
+		#{$base}__content-wrapper {
+			transform: translate(0, -100%);
+			margin-inline-start: $wikit-Popover-distance + $wikit-Popover-pointer-height;
+			margin-block-start: $wikit-Popover-pointer-width;
+			inset-inline-start: 100%;
+		}
+		#{$base}__pointer {
+			width: $wikit-Popover-pointer-width / 2;
+			height: $wikit-Popover-pointer-height * 4;
+			inset-block-start: $wikit-Popover-pointer-width + $wikit-Popover-pointer-width / 2;
+			top: $wikit-Popover-pointer-width + $wikit-Popover-pointer-height;
+			inset-inline-start: -$wikit-Popover-pointer-height;
+		}
+
+		#{$base}__pointer::before {
+			inset-block-end: auto;
+		}
+	}
+	&--right-bottom{
+		#{$base}__content-wrapper {
+			transform: translate(0, -50%);
+			margin-inline-start: $wikit-Popover-distance + $wikit-Popover-pointer-height;
+			inset-inline-start: 100%;
+		}
+		#{$base}__pointer {
+			width: $wikit-Popover-pointer-width / 2;
```

Comment:
No. This requires you knowing what the actual ratio between width and height is. Rather, when it is rotated by 90 degrees, then the width becomes the height and vice versa, and we should use the values accordingly.